src/relay/blockchain/node.py

```diff
@@ -2,6 +2,9 @@
 import os
 from collections import namedtuple
 
+from tldeploy.identity import MetaTransactionStatus
```

Comment:
I dont like this dependency. It does not make sense that normal transaction depend on MetaTransactionStatus. Just redefine it here as `TransactionStatus`